src/autogluon/cloud/cluster/cluster_config_generator.py

```diff
@@ -1,19 +1,65 @@
+import os
 from abc import ABC, abstractmethod
-from typing import Any, Dict, Union
+from typing import Any, Dict, Optional, Union
+
+import yaml
+
+DEFAULT_CONFIG_LOCATION = os.path.join(os.path.dirname(__file__), "..", "default_cluster_configs")
 
 
 class ClusterConfigGenerator(ABC):
-    @abstractmethod
-    @staticmethod
-    def get_default_config() -> Dict[str, Any]:
+    default_config_file = os.path.join(DEFAULT_CONFIG_LOCATION, "DUMMY")
```

Comment:
Not sure we should use `DUMMY` value for the default location; can be `DEFAULT_CONFIG` or something else neutral and descriptive.